cli/module_build.go

```diff
@@ -453,6 +455,11 @@ func reloadModuleAction(c *cli.Context, vc *viamClient) error {
 				part.Part.Fqdn, c.Bool(debugFlag), false, false, []string{manifest.Build.Path},
 				reloadingDestination(c, manifest), logging.NewLogger("reload"))
 			if err != nil {
+				if s, ok := status.FromError(err); ok && s.Code() == codes.PermissionDenied {
+					warningf(c.App.ErrWriter, "RDK couldn't write to the default file copy destination. "+
+						"If you're running as non-root, try adding --home $HOME or --home /user/username to your CLI command. "+
+						"Alternatively, run the RDK as root.")
+				}
```

Comment:
I wonder if `errors.Is(err, fs.ErrPermission){}` works across gRPC boundaries...

But this err pattern is also clean.

Reply:
guessing it sends a string? would be neat if something got preserved